src/Plugin/facets/facet_source/ListFacetSource.php

```diff
@@ -58,7 +59,8 @@ public function __construct(array $configuration, $plugin_id, array $plugin_defi
     $this->currentPathStack = $current_path_stack;
     $this->requestStack = $request_stack;
     $this->entityTypeManager = $entity_type_manager;
-    $this->index = $plugin_definition['index'];
+    // This needs to be loaded as search api doesn't rely on getIndex().
+    $this->index = $this->entityTypeManager->getStorage('search_api_index')->load($plugin_definition['index']);
```

Comment:
Why are you doing this? Not all the times this plugin needs to be instantiated will you need the index loaded. So load the index only when you need it. It was correct before.

Reply:
Search API does not always use the getIndex method to access this property..